src/styles/constants/theme-print.scss

```
@@ -1,10 +1,11 @@
 @page print-portrait {
 	size: A4 portrait;
+	margin: 10mm;
 }
 
 @page print-landscape {
 	size: A4 landscape;
```

Stylelint (17.13.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)